docs/course/index.md

```diff
@@ -130,11 +130,7 @@ This file specifies basic information about the course:
     }
     ```
 
-??? note "Format specification for `infoCourse.json`"
-
-    ```json
-    --8<-- "apps/prairielearn/src/schemas/schemas/infoCourse.json"
-    ```
+!!! info "[`infoCourse.json` reference](../schemas/infoCourse.md) :robot:"
```

Comment:
I don't love the "Test" link, and I don't think it's that useful for UUIDs. Can we disable that?

<img width="799" alt="Screenshot 2025-03-27 at 14 18 07" src="https://github.com/user-attachments/assets/5160877d-eb0e-4afb-b2c6-1822f21996e4" />

Reply:
Hmm, it's a little bit of a hassle to disable it. Is this a nit or something you absolutely want disabled? Maybe different styling (link to regex101 on `code block` would look nicer?

Reply:
I can't possibly imagine anyone using this. If you must have it, sure, a link on the regexp itself would work? Just seems wholly unnecessary and potentially confusing (why should this link to something? is it important? I won't know til I click it, and when I do, I'll discover that no, it's not important).

Reply:
Removed in [0b45b86](https://github.com/PrairieLearn/PrairieLearn/pull/11615/commits/0b45b86df221107fa3bb54c59910ff2173cd2d6c).